docs/operators/regexparser.md

```diff
@@ -0,0 +1,129 @@
+
+**Note**: field names of POJO must match field names in schema and in the same order as it appears in the incoming data.
+
+## Overview
+The operator is **idempotent**, **fault-tolerant** and **partitionable**.
+
+## Operator Information
+1. Operator location:**_malhar-contrib_**
+2. Available since:**_3.7.0_**
+3. Operator state:**_Evolving_**
+4. Java Package:[com.datatorrent.contrib.parser.RegexParser](https://github.com/apache/apex-malhar/blob/master/contrib/src/main/java/com/datatorrent/contrib/parser/RegexParser.java)
+
+
+## <a name="props"></a>Properties of Regex Parser
+User need to set the schema string which describes data schema as well as specify the format on date fields if any.
+e.g.
+
+```xml
+{
+   "fields": [
+      {
+        "name": "date",
+        "type": "Date",
+        "constraints": {
+          "format": "yyyy:MM:dd:hh:mm:ss"
+         }
+      },
+      {
+        "name": "id",
+        "type": "Integer"
+      },
+      {
+        "name": "message",
+        "type": "String"
+      }
+   ]
+}
+```
+
+User need to set the splitRegexPattern which is the regex expression that describes the pattern of the incoming data.
+
+| **Property** | **Description** | **Type** | **Mandatory** | **Default Value** |
+| -------- | ----------- | ---- | ------------------ | ------------- |
+| *schema* | [Schema](https://github.com/apache/apex-malhar/blob/master/contrib/src/main/java/com/datatorrent/contrib/parser/DelimitedSchema.java)  describing data schema | String | YES | N/A |
+| *splitRegexPattern* | regex expression that describes the pattern of incoming data | String | YES | N/A |
+
+ 
+## Platform Attributes that influences operator behavior
+
+| **Attribute** | **Description** | **Type** | **Mandatory** |
+| -------- | ----------- | ---- | ------------------ |
+| *out.TUPLE_CLASS* | TUPLE_CLASS attribute on output port which tells operator the class of POJO which need to be emitted | Class| Yes |
+ 
+## <a name="dataTypes"></a>Supported DataTypes in Schema
+  * Integer
+  * Long
+  * Double
+  * Character
+  * String
+  * Boolean
+  * Date
+  * Float
+
+## <a name="constraints"></a>Schema Constraints 
+|**DataType**| **Constraints** | **Description** |
+| -------- | -------- | -------- |
+| *Date* | format | A simple date format as specified in the SimpleDateFormat class: http://docs.oracle.com/javase/8/docs/api/java/text/SimpleDateFormat.html?is-external=true |
```

Comment:
Please format the link in markdown and verify that it is rendered properly with Chrome and mkdocs